tests/ai_safety/trustyai_service/service/multi_ns/conftest.py

```
@@ -153,25 +131,21 @@ def mlserver_runtime_multi_ns(admin_client, model_namespaces) -> Generator[list[
     model_namespaces: list[Namespace],
-    minio_pod: Pod,
-    minio_service: Service,
-    minio_data_connection_multi_ns: list[Secret],
     mlserver_runtime_multi_ns: list[ServingRuntime],
     kserve_raw_config: ConfigMap,
     kserve_logger_ca_bundle_multi_ns: list[ConfigMap],
 ) -> Generator[list[InferenceService], Any]:
     with ExitStack() as stack:
         models = []
-        for ns, secret, runtime in zip(model_namespaces, minio_data_connection_multi_ns, mlserver_runtime_multi_ns):
+        for ns, runtime in zip(model_namespaces, mlserver_runtime_multi_ns):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)